lib/streamlit/util.py

```diff
@@ -245,26 +245,25 @@ def open_browser(url):
 
     system = platform.system()
 
+    if system == "Windows":
+        # Treat Windows separetely because:
+        # 1. /dev/null doesn't exist.
+        # 2. subprocess.Popen(['start', url]) doesn't actually pop up the
+        #    browser even though 'start url' works from the command prompt.
+        # Fun!
+        import webbrowser
+        webbrowser.open(url)
+        return
```

Comment:
Changed this code because we no longer have to append "index.html" to URLs on Windows.

I also moved the Windows `if` block into its own thing at the top to make it very clear that the function ends there when on Windows.